assets/templates/dining_lists/dining_day.html

```diff
@@ -72,10 +72,32 @@ <h4 class="my-0 {% if date.upcoming == date %}font-weight-bold{% endif %}">
             {% include 'dining_lists/snippet_diningslot.html' %}
         {% endfor %}
     </div>
-    {% if can_create_slot %}
-        <a href="{% url 'new_slot' year=date.year month=date.month day=date.day %}"
-           class="col-12 btn btn-block btn-primary">Create and claim a new slot</a>
-    {% endif %}
+
+    <div class="row">
+        <div class="btn-group col-12 d-md-inline-flex d-none">
+            <a class="btn btn-secondary col-3"
+               href="{% url 'credits:transaction_donate' %}?redirect={{ request.path }}">
+                <i class="fas fa-coins"></i> Single use
```

Comment:
I don't think it's clear for a user what this is for. Maybe use something like 'Kitchen use without creating dining list'. Or you could add some explanation below in small font.

Reply:
The gist is: the user should not need to go to the help page to have UI actions explained